functions/system.bash

```diff
@@ -554,3 +554,31 @@ prepare_serial_port() {
 
   whiptail --title "Operation successful" --msgbox "$successText" 9 80
 }
+
+
+## Function for installing the ELElabs EZSP firmware update utility
+## Useful to flash various ZigBee controllers that have an Ember chipset.
+##
+##    ezspUtility_setup
+##
+##
+ezspUtility_setup() {
+  local repo=https://github.com/Elelabs/elelabs-zigbee-ezsp-utility
+  local destdir=/usr/share/openhab
+  local target=Elelabs_EzspFwUtility.py
+  local successText="The ELElabs python tool to flash Ember chipset ZigBee controllers has been successfully installed as ${destdir}/${target}.\\n\\nSee https://github.com/Elelabs/elelabs-zigbee-ezsp-utility#how-to how to use it."
+
+
+  temp="$(mktemp -d "${TMPDIR:-/tmp}"/openhabian.XXXXX)"
+
+  echo -n "$(timestamp) [openHABian] Installing ELElabs firmware flash tool ... "
+  cond_redirect git clone "$repo" "$temp"
+  git clone https://github.com/xsp1989/zigbeeFirmware.git /tmp/zbfw
+  if ! cond_redirect pip3 install -r ${temp}/requirements.txt; then echo "FAILED (install .py)"; return 1; fi
```

Comment:
```suggestion
  if ! cond_redirect pip3 install -r ${temp}/requirements.txt; then echo "FAILED (install python requirements)"; return 1; fi
```

Reply:
ok
(btw I'd take the suggestion, however I don't know what happens when I batch-fix all of yout raised comments in my local sources...  I believe accepting a suggestion will patch my online repo. But that means that I would need to merge (pull ?) those changes into my local sources before I can proceed there.)  But I'm not sure what'll happen so that's why I didn't dare to accept suggestions so far.

Reply:
I do the same, GitHub needs to work on that part.